pkg/sqlite/query.go

```diff
@@ -1117,3 +1117,71 @@ func (s *SQLQuerier) GetPropertiesForBundle(ctx context.Context, name, version,
 
 	return
 }
+
+func (s *SQLQuerier) GetTailForBundleInChannel(ctx context.Context, csvName string) (bundleNames []string, err error) {
+	getDepthQuery := `SELECT DISTINCT channel_name, package_name, depth FROM channel_entry WHERE operatorbundle_name=?`
+
+	getTailQuery := `SELECT DISTINCT operatorbundle_name FROM channel_entry 
+				WHERE channel_name=?
+				AND package_name=?
+				AND depth > ?`
```

Comment:
Without looking it up, I don't know if this is the right direction for `depth`. A unit test to prove it's correct would help 😄